README.md

```diff
@@ -18,10 +18,4 @@ Alicerce to your `Cartfile`:
 
 ```
 github "Mindera/Alicerce"
-```
-
-# To Do
-- [ ] Add Network Stack
-- [ ] Add Memory Persistence
-- [ ] Add Disk Memory Persistence
-- [ ] Add Logger
\ No newline at end of file
+```
```

Comment:
Perhaps in the future we could add a feature list with this stuff (and examples) 😉